<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="info_label1">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>21</y>
      <width>114</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ви увійшли як : </string>
    </property>
   </widget>
   <widget class="QLabel" name="info_label2">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>21</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>___</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="hr_w">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>1051</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="contractors">
     <attribute name="title">
      <string>Контрагенти</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="interview">
     <attribute name="title">
      <string>Співбесіда</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="position">
     <attribute name="title">
      <string>Посада</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="staff">
     <attribute name="title">
      <string>Персонал</string>
     </attribute>
     <zorder>manag_w</zorder>
     <zorder>manag_w</zorder>
     <zorder>manag_w</zorder>
    </widget>
   </widget>
   <widget class="QPushButton" name="exit_main_button">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>460</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="manag_w">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>941</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="goods">
     <attribute name="title">
      <string>Товари</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="sales">
     <attribute name="title">
      <string>Продаж</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="purchase">
     <attribute name="title">
      <string>Закупка</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1073</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
